astropy/io/ascii/tdat.py

```diff
@@ -376,10 +469,10 @@ def write(self, lines):
                 "'table_name' must be specified\n"  # noqa: ISC003
                 + f"{_STD_MSG}\n"
                 + "This should be specified in the Table.meta.\n"
-                + "default value of 'astropy_table' being assigned.",
+                + "default value of 'origin_astropy' being assigned.",
                 TdatFormatWarning,
             )
-            lines.append("table_name = astropy_table")
+            lines.append("table_name = origin_astropy")
```

Comment:
As per my comment on the intent of `origin` in the file format documentation, I suggest changing this back to `astropy_table` throughout.